sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/README.md

```diff
@@ -1,25 +1,305 @@
-# Azure Tables client library for .NET
+# Azure WebJobs Tables client library for .NET
+
+This extension provides functionality for accessing Azure Tables in Azure Functions.
 
 ## Getting started
 
 ### Install the package
 
+Install the Storage Blobs extension with [NuGet][nuget]:
+
+```dotnetcli
+dotnet add package Azure.WebJobs.Extensions.Tables
```

Comment:
We are not using the standard pattern?

Reply:
`.Data.` felt like a somewhat useless prefix but it's up for discussion.

Reply:
no I meant the Microsoft.Azure.WebJobs pattern.

Reply:
Ah, missed that. Good point.